### ISSUE TITLE
Term Request: Granular corneal stromal deposits

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11745]
**Preferred term label:**
Granular corneal stromal deposits

**Synonyms**
* Crumb-like corneal opacities
* Snowflake corneal stromal opacities
* Hyaline corneal stromal granules

**Definition (free text, please give PubMed ID)**
Discrete, sharply demarcated, grayish-white granular opacities (resembling breadcrumbs, glassy splinters, or snowflakes) located within the central anterior-to-deep stroma with clear intervening stroma, staining brilliant red with Masson trichrome (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Opacification of the corneal stroma (HP:0007759)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Granular corneal dystrophy, type 1 (GCD1) (ORPHA:98962, OMIM:121900)
* Granular corneal dystrophy, type 2 (GCD2) (ORPHA:98963, OMIM:607541)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
